docs/docs_mobile_app/polityka-prywatnosci-klient.md

```diff
@@ -0,0 +1,184 @@
+# POLITYKA PRYWATNOŚCI
+
+## Aplikacji mobilnej MeldMoto dla Klientów
+
+**Spis treści:**
+* § 1 Informacje ogólne
+* § 2 Administrator danych osobowych
+* § 3 Pozyskiwanie danych i cel ich przetwarzania
+* § 4 Kategorie danych osobowych
+* § 5 Odbiorcy danych osobowych
+* § 6 Archiwizacja danych osobowych
+* § 7 Uzyskiwanie dostępu i aktualizacja danych osobowych, reklamacje
+* § 8 Przetwarzanie danych osobowych w sposób zautomatyzowany
+* § 9 Geolokalizacja
+* § 10 Zmiany Polityki prywatności
+
+## § 1 INFORMACJE OGÓLNE
+
+Polityka prywatności Aplikacji mobilnej MeldMoto dla Klientów nie stanowi źródła obowiązków dla osób korzystających z Aplikacji. Ma ona charakter wyłącznie informacyjny, nie jest umową ani regulaminem.
+
+Wszystkie wyrażenia i słowa zapisywane za pomocą wielkiej litery (np. Aplikacja, Warsztat, Klient itd.) należy rozumieć zgodnie z [Regulaminem dla Klientów](https://meldmoto.github.io/terms_and_conditions/regulamin-klient).
+
+W przypadku zaistnienia jakichkolwiek rozbieżności pomiędzy niniejszą Polityką prywatności a zgodami na przetwarzanie danych osobowych, udzielonymi przez Klienta, podstawą prawną do określenia zakresu działań Administratora są dobrowolnie wyrażone zgody lub przepisy prawa, które znajdują zastosowanie w danej sytuacji faktycznej.
+
+## § 2 ADMINISTRATOR DANYCH OSOBOWYCH
+
+Administratorem Twoich danych osobowych jest Wojciech Ozimek, wykonujący działalność gospodarczą pod firmą MeldMoto Wojciech Ozimek, z siedzibą w Krakowie przy ulicy Skalica 18, wpisany do Centralnej Ewidencji i Informacji o Działalności Gospodarczej, NIP: 6762596425, REGON: 388840860.
+
+We wszelkich kwestiach związanych z ochroną danych osobowych zachęcam do kontaktu na wyżej wskazany adres lub za pośrednictwem adresu e-mail: <hello@meldmoto.pl>.
```

Comment:
remove `<` and `>` around emails

Could you also try adding the email between ` marks?

Reply:
(not sure about ` how will work to be honest)